src/cuda/gpu_oei.h

```diff
@@ -149,7 +149,7 @@ __device__ void iclass_oei(unsigned int I, unsigned int J, unsigned int II, unsi
         // get Xcoeff, which is a product of overlap prefactor and contraction coefficients 
         QUICKDouble Xcoeff_oei = LOC4(devSim.Xcoeff_oei, kStartI+III, kStartJ+JJJ, I - devSim.Qstart[II], J - devSim.Qstart[JJ], devSim.jbasis, devSim.jbasis, 2, 2);
 
-        if(abs(Xcoeff_oei) > devSim.integralCutoff){
+        if(abs(Xcoeff_oei) > DEFAULT_INTEGRAL_CUTOFF){
```

Comment:
This should use `quick_method%coreIntegralCutoff` to be in line with the CPU code and to retain ability to change the cutoff.

Reply:
Fixed.